pkg/runtime/reconciler.go

```diff
@@ -1064,6 +1042,50 @@ func (r *resourceReconciler) getEndpointURL(
 	return r.cfg.EndpointURL
 }
 
+// getResyncPeriod returns the duration a reconciler should use to drift remediate custom
```

Comment:
Clearer to say "the period of the recurring reconciler process which remediates drift in custom resources" here.   Generally, prefer the term "period" over "duration" here.